.ci/prepare_release

```diff
@@ -6,33 +6,14 @@
 
 set -e
 
-
-# add full-fleshed tar and helm
-BASE_URL="https://get.helm.sh"
-HELM_VERSION=v2.17.0
-TAR_FILE="helm-${HELM_VERSION}-linux-amd64.tar.gz"
-
-apk add --update --no-cache curl ca-certificates tar 
-curl -L ${BASE_URL}/${TAR_FILE} |tar xvz 
-mv linux-amd64/helm /usr/bin/helm 
-chmod +x /usr/bin/helm 
-
-
 if [[ -z "${SOURCE_PATH}" ]]; then
   export SOURCE_PATH="$(readlink -f "$(dirname ${0})/..")"
 else
   export SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
 fi
 
-## currently disabled, as controller registration happens per shoot by extension-shoot-cert-service
-#"${SOURCE_PATH}/hack/generate-controller-registration.sh" \
-#  cert-management \
-#  "${SOURCE_PATH}/charts/cert-management/" \
-#  "${SOURCE_PATH}/examples/gardener-controllerregistration.yaml" \
-#  Issuer:gardener
-
 VERSION_FILE="$(readlink -f "${SOURCE_PATH}/VERSION")"
 VERSION="$(cat "${VERSION_FILE}")"
 VERSIONTAG="${VERSION//-dev/-master}"
 
-sed -i -e "s/  tag: .*/  tag: ${VERSIONTAG}/" "${SOURCE_PATH}/charts/cert-management/values.yaml"
+sed -i -e "s/newTag: .*/newTag: ${VERSIONTAG}/" "${SOURCE_PATH}/manifests/profiles/example/kustomization.yaml"
```

Comment:
Should we rather edit the deployment.yaml?

Reply:
The deployment.yaml is generated and has the static tag `latest`. Looks not suitable to me.